editors/sc-ide/core/sc_introspection.cpp

```diff
@@ -202,14 +202,25 @@ bool Introspection::ensureIntrospectionData() const {
 
 const Class* Introspection::findClass(const QString& className) const {
     if (!ensureIntrospectionData())
-        return NULL;
+        return nullptr;
 
-    ClassMap::const_iterator klass_it = mClassMap.find(className);
-    if (klass_it == mClassMap.end()) {
+    auto classIterator = mClassMap.find(className);
+
+    if (classIterator == mClassMap.end()) {
+        return nullptr;
+    }
+
+    return classIterator->second.data();
```

Comment:
generally i find `return it == map.end() ? nullptr : it->second.data()` to be just as clear while saving a couple lines, but this is also fine.